data/transactions/logic/assembler.go

```diff
@@ -1143,6 +1173,49 @@ func (ops *OpStream) assemble(fin io.Reader) error {
 	return nil
 }
 
+func (ops *OpStream) pragma(line string) error {
```

Comment:
Is is the same as in master? maybe back merge master into teal3 to reduce the diff?

Reply:
It is.  I had to pull it in so that I could get access to the `make integration` fixes so I could right v24 tests.